R/run_stress_test.R

```diff
@@ -127,53 +124,35 @@ run_stress_test <- function(asset_type,
   cat("-- Exported results to designated output path. \n")
 }
 
-#' Iterate over stress test runs
-#'
-#' @param n Numeric.
-#' @param args_tibble  A tibble holding a set of params for
-#'   `run_stress_test_imp` per row.
-#'
-#' @return List of stress test results.
```

Comment:
Internal functions don't need documentation and in this case its most maintainable to not do so.